### ISSUE TITLE
problems with utils/yamlio tests

### ISSUE THREAD

**Issue by bridwell** [UDST/urbansim#188]
Has anyone else experienced problems with these tests? Mine seem to be failing.

**Comment by bridwell** [comment]
The issue seems to be with the line separator. This is py27 on windows 10.

**expected_yaml:** 
'name: test\r\n\r\nytransform: xyz\r\n\r\nunordered: abc\r\n'

**result from yamlio.ordered_yaml:**
'name: test\n\r\nytransform: xyz\n\r\nunordered: abc\n'

**Comment by pksohn** [comment]
Thanks for finding this. It might be that we want to replace `os.linesep` with a basic `\n` as according to [this](https://docs.python.org/2/library/os.html#os.linesep). I'll look into it further. We're hoping to get Windows-based continuous integration up and running which should hopefully catch some of these cross-platform issues.

**Comment by bridwell** [comment]
This seems to work for me, but I'm not sure if this works on other platforms:

```python
@pytest.fixture
def expected_yaml():
    return (
        'name: test{carriage}'
        'ytransform: xyz{carriage}'
        'unordered: abc{newline}').format(carriage='\n\r\n', newline='\n')
```

**Comment by bridwell** [comment]
The other issue I'm seeing is the index data types are changing from int32 to in64, when loading from yaml. This is causing any reference to ```pdt.assert__series_equal``` to fail. Replacing the assertions to only compare values seems to take care of this:

```python

# replace references to pdt.assert_series_equal with this:
def assert_series_equal(a, b):
    assert (a.index.values == b.index.values).all()
    assert (a.values == b.values).all()

# replace references to pdt.assert_frame_equal with this:
def assert_dfs_equal(a, b):
    assert (a.columns == b.columns).all()
    assert (a.index.values == b.index.values).all()
    assert (a.values == b.values).all()
```

After making this change, and the string formatting change above, the yamlio tests pass.

**Comment by pksohn** [comment]
Scott, FYI I replicated your errors in the Appveyor CI environment in the new `appveyor` branch: https://ci.appveyor.com/project/pksohn/urbansim.

Do you mind merging the `appveyor` branch into yours, then submitting a PR? That would then trigger automated builds/tests in both Linux and Windows.

**Comment by bridwell** [comment]
I think the PR is ready. This passes in both environments now.

References to os.linesep in tests and yamlio have been replaced with '/n' per: 
https://docs.python.org/2/library/os.html?highlight=linesep#os.linesep

**Comment by pksohn** [comment]
Fixed with #190.
